docs/install/from_source.rst

```diff
@@ -315,7 +332,7 @@ configuration. A workaround for this is to do the following commands:
 
         brew install openblas gfortran
 
-        pip install pybind11 cython pythran  
+        pip install pybind11 cython pythran
```

Comment:
Space error.

Reply:
I have a vim plugin that automatically removes trailing spaces.  So IIUC this is actually fixing a space error.